### PR TITLE
README fixes for 2.3

### PR DESCRIPTION
**Pull request by Holzhaus**
Probably a good idea to clean this up a little before tagging 2.3.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,42 +24,27 @@ Have a bug or feature request? [File a bug on Launchpad][fileabug].
 Want to get involved in Mixxx development? Assign yourself a bug from the [easy
 bug list][easybugs] and get started!
 
-## Compiling on Linux
-    $ mkdir build
-    $ cd build
-    $ cmake ..
-    $ cmake --build .
-Please see our helpful guide on the [wiki] for more information: [Compiling on Linux]
+## Building Mixxxx
```
Comment (ninomp) [inline README.md:27]: Is Mixxx spelled with four 'x' deliberately here or is this a typo?

Comment (Holzhaus) [reply README.md:27]: typo, thanks for pointing that out.

**Review by Swiftb0y — COMMENTED** [review]
@daschuer do you have an opinion regarding removing the plain README file? Does it still have a purpose which isn't fulfilled by the markdown version?

**Comment by daschuer** [comment]
I don't think so. It is OK for me to remove the plain text version.

**Review by Swiftb0y — APPROVED** [review]
Merge?

**Comment by Holzhaus** [comment]
> Merge?

Do it. 😁

**Pull request merged**